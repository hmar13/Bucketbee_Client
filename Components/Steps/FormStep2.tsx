import React, { useState, Dispatch, SetStateAction } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import * as Animatable from 'react-native-animatable';
import { Button } from 'react-native-paper';
import RadioGroup, { RadioButtonProps } from 'react-native-radio-buttons-group';
import theme from '../../styles/theme.style';

const radioButtonsData = [
  {
    id: '1', // acts as primary key, should be unique and non-empty string
    label: 'Beach',
    value: 'option1',
    labelStyle: {
      color: theme.PRIMARY_COLOR_XLITE,
      fontSize: 20,
      fontFamily: 'Lato_300Light',
    },
    color: theme.PRIMARY_COLOR_XLITE,
    size: 20,
    containerStyle: {
      alignSelf: 'flex-start',
    },
  },
  {
    id: '2',
    label: 'Mountains',
    value: 'option2',
    color: theme.PRIMARY_COLOR_XLITE,
    labelStyle: {
      color: theme.PRIMARY_COLOR_XLITE,
      fontSize: 20,
      fontFamily: 'Lato_300Light',
    },
    size: 20,
  },
  {
    id: '3',
    label: 'City',
    value: 'option3',
    color: theme.PRIMARY_COLOR_XLITE,
    size: 20,
    labelStyle: {
      color: theme.PRIMARY_COLOR_XLITE,
      fontSize: 20,
      fontFamily: 'Lato_300Light',
    },
  },
];

type SetCountType = Dispatch<SetStateAction<number>>
type SetVibeType = Dispatch<SetStateAction<string | undefined>>


interface Props {
  setCount: SetCountType;
  setVibe: SetVibeType;
  vibe: string | undefined;
}

const FormStep2: React.FC<Props> = ({ setCount, setVibe, vibe }) => {
  const [radioButtons, setRadioButtons] = useState<RadioButtonProps[]>(
    radioButtonsData,
  );

  const onPressRadioButton = (radioButtonsArray: RadioButtonProps[]) => {
    setRadioButtons(radioButtonsArray);
  };

  const handleSubmit = (radioButtons: RadioButtonProps[]) => {
    const selected = radioButtons.filter((b) => b.selected === true);
    if (selected[0].label) {
      setVibe(selected[0].label);
    }

    setCount((val: number) => val + 1);
  };



  return (
    <View style={styles.container}>
      <Animatable.View animation="fadeInUpBig" style={styles.wrapper}>
        <Text style={styles.text}>Beach / mountain / city?</Text>
        <RadioGroup
          layout="row"
          radioButtons={radioButtons}
          onPress={onPressRadioButton}
        />
        <Button
          style={styles.button}
          labelStyle={{ color: theme.PRIMARY_COLOR }}
          onPress={() => handleSubmit(radioButtons)}
        >
          NEXT
        </Button>
      </Animatable.View>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: theme.PRIMARY_COLOR,
  },
  wrapper: {
    height: 240,
    width: 300,
    justifyContent: 'space-around',
  },
  input: {
    width: 280,
    height: 50,
    borderBottomColor: theme.PRIMARY_COLOR_XLITE,
    borderBottomWidth: 2,
    color: theme.PRIMARY_COLOR_XLITE,
    fontSize: 22,
    fontFamily: 'Lato_300Light',
  },
  button: {
    backgroundColor: theme.PRIMARY_COLOR_XLITE,
    marginVertical: 20,
    color: theme.PRIMARY_COLOR,
    height: 50,
    width: 100,
    justifyContent: 'center',
  },
  text: {
    color: theme.PRIMARY_COLOR_XLITE,
    fontSize: 24,
    fontFamily: 'Poppins_400Regular',
    paddingBottom: 15,
  },
});

export default FormStep2;
