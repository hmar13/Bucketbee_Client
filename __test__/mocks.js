import theme from '../styles/theme.style';

const bucket = {
  __typename: 'Bucket',
  categories: [{
    __typename: 'Category',
    id: '604ca4e87f7d89153b39d99f',
    label: "Hello",
    places: [],
  }],
  date_created: '2021-03-13T11:41:28.273Z',
  id: "604ca4e87f7d89153b39d9a0",
  notes: "Hello",
  title: "Hello",
}
const userA = {
  _typename: 'User',
  birthday: null,
  createdAt: '2021-03-12T12:05:53.895Z',
  email: 'A',
  emojis: ':) ;) :O',
  firstName: 'A',
  friends: [{
    __typename: "User",
      birthday: null,
      createdAt: "2021-03-11T17:25:27.762Z",
      email: 'B',
      firstName: 'B',
      id: '604a52875a22a656d698c988',
      lastName: null,
      username: 'B',
  }],
  id: '604b592130793c1bff327555',
  lastName: null,
  location: 'London',
  profile_pic: null,
  updatedAt: '2021-03-15T09:02:58.715Z',
  username: 'A',
  vibe: 'Beach'
}
const userB = {
  _typename: 'User',
  birthday: null,
  createdAt: '2021-03-12T12:10:53.895Z',
  email: 'B',
  emojis: ':) :O :P',
  firstName: 'B',
  friends: [{
    __typename: "User",
      birthday: null,
      createdAt: "2021-03-12T12:05:53.895Z",
      email: 'A',
      firstName: 'A',
      id: '604b592130793c1bff327555',
      lastName: null,
      username: 'A',
  }],
  id: '604a52875a22a656d698c988',
  lastName: null,
  location: 'London',
  profile_pic: null,
  updatedAt: '2021-03-15T09:02:58.715Z',
  username: 'B',
  vibe: 'Beach'
}

const place = {
  __typename: 'Place',
  description: 'locality, political',
  formatted_address: 'London, UK',
  id: '',
  imgArr: [
    'https://lh3.googleusercontent.com/p/AF1QipOmH4cmfDXKSZsY0ECNfEVo99-sJ9JiwW0hOE5y=s1600-h619',
    'https://lh3.googleusercontent.com/p/AF1QipPlfRNdjQbAgVXJ6CO8xm6ArPNtIyIK-SqHx4V1=s1600-h911',
    'https://lh3.googleusercontent.com/p/AF1QipMGBhjSmSICwYc2uMiFQ0l-bZm-Yx69est4gBvO=s1600-h1200',
    'https://lh3.googleusercontent.com/p/AF1QipMx47l1JKY-Nj9p6u5yFIj-TI7ubxM6m1_rdwSH=s1600-h1200',
    'https://lh3.googleusercontent.com/p/AF1QipO1gRyYKC04M2jwJh6mSeuDvzhXqks1BJZW7_Yw=s1600-h1200',
    'https://lh3.googleusercontent.com/p/AF1QipO5rYsQ8pjrN9Tmspu1uSKtLbyy4wZvO_bMyzH1=s1600-h1200',
    'https://lh3.googleusercontent.com/p/AF1QipNB7-V6PQypu5RSJnazhue3BNaHtiBA5bqqYI5Z=s1600-h1200',
    'https://lh3.googleusercontent.com/p/AF1QipNk5QU10zjui5OrIQ2WuMLuWChDJDmu0sGz-Zi7=s1600-h1200',
    'https://lh3.googleusercontent.com/p/AF1QipMRx0je916q6jCTwBi3lU5AcWNHIa4dHa91eDZq=s1600-h1200',
    'https://lh3.googleusercontent.com/p/AF1QipNKpEuPRT3zlCBdgjvK5zq1-LB1TlcOCNMAyhCl=s1600-h1200',
  ],
  international_phone_number: undefined,
  latitude: 51.5073509,
  longitude: -0.1277583,
  name: 'London',
  notes: '',
  open_now: undefined,
  rating: undefined,
  review: undefined,
  url: 'https://maps.google.com/?q=London,+UK&ftid=0x47d8a00baf21de75:0x52963a5addd52a99',
  user_ratings_total: undefined,
  weekday_text: undefined,
}

const category = {
  __typename: 'Category',
  id: '604b597030793c1bff327556',
  label: 'Music',
  places: [place],
}

const message = {
  __typename: 'Message',
  author: '604b592130793c1bff327555',
  content: 'TEST TEST TEST',
  createdAt: '2021-03-15T09:05:56.583Z',
  description: 'message',
  id: '604f23740ea199065c8a602f',
  photo: null,
  timeslots: []
}
const chat = {
  __typename: 'Chat',
  admin: null,
  createdAt: '2021-03-15T09:05:50.079Z',
  id: '604f236e0ea199065c8a602e',
  members: [userA, userB],
  messages: [message],
  name: 'B',
  updatedAt: '021-03-15T16:05:48.012Z',
}

const radioButtons = [
  {
    id: '1', // acts as primary key, should be unique and non-empty string
    label: 'Beach',
    value: 'option1',
    labelStyle: {
      color: theme.PRIMARY_COLOR_XLITE,
      fontSize: 20,
      fontFamily: 'Lato_300Light',
    },
    color: theme.PRIMARY_COLOR_XLITE,
    size: 20,
    containerStyle: {
      alignSelf: 'flex-start',
    },
  },
  {
    id: '2',
    label: 'Mountains',
    value: 'option2',
    color: theme.PRIMARY_COLOR_XLITE,
    labelStyle: {
      color: theme.PRIMARY_COLOR_XLITE,
      fontSize: 20,
      fontFamily: 'Lato_300Light',
    },
    size: 20,
  },
  {
    id: '3',
    label: 'City',
    value: 'option3',
    color: theme.PRIMARY_COLOR_XLITE,
    size: 20,
    labelStyle: {
      color: theme.PRIMARY_COLOR_XLITE,
      fontSize: 20,
      fontFamily: 'Lato_300Light',
    },
    selected: true,
  },
];

module.exports = { bucket, userA, userB, place, category, message, chat, radioButtons }