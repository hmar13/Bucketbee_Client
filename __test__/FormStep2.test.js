import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from 'react-native-paper';
import RadioGroup from 'react-native-radio-buttons-group';
import { shallow } from 'enzyme';
import FormStep2 from '../Components/Steps/FormStep2';
import { radioButtons } from './mocks';


describe('<FormStep2 />', () => {
  const setCount = jest.fn();
  const setVibe = jest.fn();
  const wrapper = shallow(<FormStep2
    setCount={setCount}
    setVibe={setVibe}
  />);

  test('Renders correctly', () => {
    const tree = renderer.create(<FormStep2 />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Should render the Text Element', () => {
    expect(wrapper.find('Text').at(0).props().children).toEqual('Beach / mountain / city?')
  });

  test('Should call setStates (Count, Vibe)', () => {
    wrapper.find(RadioGroup).simulate('press', radioButtons);
    wrapper.find(Button).simulate('press');

    expect(setCount).toHaveBeenCalled();
    expect(setVibe).toHaveBeenCalled();
  });

});
