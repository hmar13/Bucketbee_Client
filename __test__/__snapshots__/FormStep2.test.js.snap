// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormStep2 /> Renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0d525f",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    forwardedRef={[Function]}
    style={
      Object {
        "height": 240,
        "justifyContent": "space-around",
        "opacity": 0,
        "transform": Array [
          Object {
            "translateY": 500,
          },
        ],
        "width": 300,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#9affdc",
          "fontFamily": "Poppins_400Regular",
          "fontSize": 24,
          "paddingBottom": 15,
        }
      }
    >
      Beach / mountain / city?
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "marginHorizontal": 10,
                "marginVertical": 5,
              },
              Object {
                "flexDirection": "row",
              },
              Object {
                "opacity": 1,
              },
              Object {
                "alignSelf": "flex-start",
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
                Object {
                  "borderColor": "#9affdc",
                  "borderRadius": 10,
                  "borderWidth": 2,
                  "height": 20,
                  "width": 20,
                },
              ]
            }
          />
          <Text
            style={
              Array [
                Object {
                  "marginLeft": 10,
                },
                Object {
                  "color": "#9affdc",
                  "fontFamily": "Lato_300Light",
                  "fontSize": 20,
                },
              ]
            }
          >
            Beach
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "marginHorizontal": 10,
                "marginVertical": 5,
              },
              Object {
                "flexDirection": "row",
              },
              Object {
                "opacity": 1,
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
                Object {
                  "borderColor": "#9affdc",
                  "borderRadius": 10,
                  "borderWidth": 2,
                  "height": 20,
                  "width": 20,
                },
              ]
            }
          />
          <Text
            style={
              Array [
                Object {
                  "marginLeft": 10,
                },
                Object {
                  "color": "#9affdc",
                  "fontFamily": "Lato_300Light",
                  "fontSize": 20,
                },
              ]
            }
          >
            Mountains
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "marginHorizontal": 10,
                "marginVertical": 5,
              },
              Object {
                "flexDirection": "row",
              },
              Object {
                "opacity": 1,
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
                Object {
                  "borderColor": "#9affdc",
                  "borderRadius": 10,
                  "borderWidth": 2,
                  "height": 20,
                  "width": 20,
                },
              ]
            }
          />
          <Text
            style={
              Array [
                Object {
                  "marginLeft": 10,
                },
                Object {
                  "color": "#9affdc",
                  "fontFamily": "Lato_300Light",
                  "fontSize": 20,
                },
              ]
            }
          >
            City
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#9affdc",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "color": "#0d525f",
          "elevation": 0,
          "height": 50,
          "justifyContent": "center",
          "marginVertical": 20,
          "minWidth": 64,
          "width": 100,
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#6200ee",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#0d525f",
                  },
                ],
              ]
            }
          >
            NEXT
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`<FormStep2 /> Should render the page without crashing 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0d525f",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    forwardedRef={[Function]}
    style={
      Object {
        "height": 240,
        "justifyContent": "space-around",
        "opacity": 0,
        "transform": Array [
          Object {
            "translateY": 500,
          },
        ],
        "width": 300,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#9affdc",
          "fontFamily": "Poppins_400Regular",
          "fontSize": 24,
          "paddingBottom": 15,
        }
      }
    >
      Beach / mountain / city?
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessible={true}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "marginHorizontal": 10,
                "marginVertical": 5,
              },
              Object {
                "flexDirection": "row",
              },
              Object {
                "opacity": 1,
              },
              Object {
                "alignSelf": "flex-start",
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
                Object {
                  "borderColor": "#9affdc",
                  "borderRadius": 10,
                  "borderWidth": 2,
                  "height": 20,
                  "width": 20,
                },
              ]
            }
          />
          <Text
            style={
              Array [
                Object {
                  "marginLeft": 10,
                },
                Object {
                  "color": "#9affdc",
                  "fontFamily": "Lato_300Light",
                  "fontSize": 20,
                },
              ]
            }
          >
            Beach
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "marginHorizontal": 10,
                "marginVertical": 5,
              },
              Object {
                "flexDirection": "row",
              },
              Object {
                "opacity": 1,
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
                Object {
                  "borderColor": "#9affdc",
                  "borderRadius": 10,
                  "borderWidth": 2,
                  "height": 20,
                  "width": 20,
                },
              ]
            }
          />
          <Text
            style={
              Array [
                Object {
                  "marginLeft": 10,
                },
                Object {
                  "color": "#9affdc",
                  "fontFamily": "Lato_300Light",
                  "fontSize": 20,
                },
              ]
            }
          >
            Mountains
          </Text>
        </View>
        <View
          accessible={true}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "alignItems": "center",
                "marginHorizontal": 10,
                "marginVertical": 5,
              },
              Object {
                "flexDirection": "row",
              },
              Object {
                "opacity": 1,
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                },
                Object {
                  "borderColor": "#9affdc",
                  "borderRadius": 10,
                  "borderWidth": 2,
                  "height": 20,
                  "width": 20,
                },
              ]
            }
          />
          <Text
            style={
              Array [
                Object {
                  "marginLeft": 10,
                },
                Object {
                  "color": "#9affdc",
                  "fontFamily": "Lato_300Light",
                  "fontSize": 20,
                },
              ]
            }
          >
            City
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#9affdc",
          "borderColor": "transparent",
          "borderRadius": 4,
          "borderStyle": "solid",
          "borderWidth": 0,
          "color": "#0d525f",
          "elevation": 0,
          "height": 50,
          "justifyContent": "center",
          "marginVertical": 20,
          "minWidth": 64,
          "width": 100,
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "disabled": undefined,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
              },
              undefined,
            ]
          }
        >
          <Text
            numberOfLines={1}
            selectable={false}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "letterSpacing": 1,
                    "marginHorizontal": 16,
                    "marginVertical": 9,
                    "textAlign": "center",
                  },
                  undefined,
                  Object {
                    "textTransform": "uppercase",
                  },
                  Object {
                    "color": "#6200ee",
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "fontFamily": "System",
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#0d525f",
                  },
                ],
              ]
            }
          >
            NEXT
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
